public/app/core/components/Page/Page.tsx

```diff
@@ -123,5 +125,12 @@ const getStyles = (theme: GrafanaTheme2) => {
       flexBasis: '100%',
       flexGrow: 1,
     }),
+    // If we have a sticky header, then we don't need to add padding to the top
+    // otherwise this will end up as too much spacing with the header
+    ifSticky: css({
+      [theme.breakpoints.up('sm')]: {
```

Comment:
I'm actually not sure whether we should have a sticky header on small screens. Since Grafana is already not strong on mobile devices this could make it even worse.
So I suggest to start using a sticky header from breakpoint lg or even xl.

Reply:
That's what I've actually [done here](https://github.com/grafana/grafana/pull/87802/files#diff-99f35e4da0dfdc1206c8481daae1e6eddee17f151461f1b484332a135aa59977R53), I'm only applying "sticky-ness" when we're above `sm` breakpoint. This here is ensuring we don't mess with the page padding unless we're `stickyHeader = true` and also greater than `sm` breakpoint